recipes/opencv/2.x/conanfile.py

```diff
@@ -193,6 +193,8 @@ def _configure_cmake(self):
         self._cmake.definitions["OPENCV_MODULES_PUBLIC"] = "opencv"
         self._cmake.definitions["BUILD_opencv_nonfree"] = self.options.nonfree
 
+        self._cmake.definitions["ENABLE_CCACHE"] = False
```

Comment:
why no `OPENCV_PYTHON_SKIP_DETECTION` for 2.x?

Reply:
disabled in opencv 2.x by https://github.com/conan-io/conan-center-index/pull/9995/commits/134884423ed3647837f7366357dfcacce90ff959